### PR TITLE
Books es6

### PR DESCRIPTION
**Pull request by eledwinn**
## Summary

1. Add Babel for compile ES6 files
2. Refactor project structure (code in src, compiled files in app)
3. Change JS code with ES6

## Screenshot
NA

## Trello Card
https://trello.com/c/IYxle3pD/20-dev-stack-ecmascript-6

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,5 +17,10 @@ tmp
...
-## Pug ##
-*.html
\ No newline at end of file
+## Pug and Babel ##
+app/*.html
+app/*/*.html
+app/*/*/*.html
+app/*.js
+app/*/*.js
+app/*/*/*.js
```
Comment (FrankIglesias) [inline .gitignore]: eol

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  'bookDetail',
+  'wInput'
+]).
+config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $urlRouterProvider) {
+  var states = [{ 
+    name: 'booksList', 
+    url: '/books', 
+    component: 'booksList',
+    resolve: {
+      books: function(Books) {
```
Comment (FrankIglesias) [inline src/app.js]: use arrow

File: src/index.pug [hunk 3]
```diff
@@ -18,4 +18,5 @@ html(lang="es" ng-app="wBooksApp" class="no-js")
     script(src="books/books.component.js")
     script(src="book-detail/book-detail.module.js")
     script(src="book-detail/book-detail.component.js")
-    script(src="app.js")
\ No newline at end of file
+    script(src="app.js")
+
```
Comment (FrankIglesias) [inline src/index.pug]: delete this line

File: src/scss/subtitle.scss [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.subtitle {
+  color: $green;
+  font-size: 1.3rem;
```
Comment (FrankIglesias) [inline src/scss/subtitle.scss]: use px

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 5]
```diff
@@ -17,5 +17,10 @@ tmp
 .DS_Store
 .idea
 
-## Pug ##
-*.html
+## Pug and Babel ##
+app/*.html
+app/*/*.html
+app/*/*/*.html
+app/*.js
```
Comment (FrankIglesias) [inline .gitignore:24]: maybe you should change this gitignore paths.... to only ingore angular istead

File: src/app.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+angular.module('wBooksApp', [
+  'ui.router',
+  'core.book',
+  'bookItem',
+  'booksList',
+  'bookDetail',
+  'wInput'
+]).
+config(['$stateProvider', '$urlRouterProvider', function($stateProvider, $urlRouterProvider) {
+  var states = [{
```
Comment (FrankIglesias) [inline src/app.js]: let

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 5 above)
Comment (eledwinn) [reply .gitignore:24]: This is ready in books-bootstrap

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**